### PR TITLE
fix celery

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
@jerith review please

File: mc2/tasks.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
-from celery import task
+from mc2 import celery_app
```
Comment (jerith) [inline mc2/tasks.py]: This module doesn't seem to exist. Should `mc2/celery.py` be `mc2/celery_app.py` instead?

Comment (jerith) [inline mc2/tasks.py]: Oh, I see now how that works. Why so convoluted?

**Comment by miltontony** [comment]
it's the new way of doing celery :(

**Comment by jerith** [comment]
Can the module be named something that isn't `celery`? Multiple modules with the same name can get very confusing.

**Comment by jerith** [comment]
Either way, :+1:

**Comment by jerith** [comment]
:+1:

**Pull request merged**